app/_hub/kong-inc/openid-connect/overview/_index.md

```diff
@@ -1902,7 +1902,7 @@ issues with the plugin or integration, try the following:
 
 1. Set Kong [log level](/gateway/latest/reference/configuration/#log_level) to `debug`, and check the Kong `error.log` (you can filter it with `openid-connect`)
    ```bash
-   KONG_LOG_LEVEL=debug kong restart
```

Comment:
This isn't faulty - it will restart Kong with config set in the environment. Running `KONG_LOG_LEVEL=debug` alone won't do anything

Reply:
@mod42 When you get a chance, could you take a look at Michael's comment above and make any changes? Thanks!